10-JavaScript/exercises/setup-linting.md

```diff
@@ -65,35 +68,25 @@ Add this file `~/.eslintrc`
 
 ## Install `Linter-Eslint`
 
-Back to Atom!  Now we can install Atom's linter and get it set up.
-
-In Atom go to preferences-->install and install `linter-eslint`.
-
-Preferences:
-
-![preferences](images/preferences.png)
-
-Install
+Head to VS Code!  Now we can install the extenstion for eslint and get it set up.
 
-![install](images/install.png)
+In VS Code open the Extensions tab and search for `ESLint`.
 
-### Atom Linter-ESLint Settings
+Extensions Tab:
 
-Then go to the settings and add these settings.
+![extensions tab](images/extensions_linter.png)
 
-Settings:
+## Settings
 
-![settings](images/search-packages.png)
+Once the new extension is installed, go to VS Code's Preferences->Settings. Open the settings JSON and add the following at the end of the file:
 
-1. .eslintrc path to: `~/.eslintrc`
-
-![.eslintrc path](images/eslintrc-path.png)
-
-2. Check `Use Global ESLint Installation`
+```json
+"eslint.options": { "configFile":".eslintrc" }
```

Comment:
I'm not sure you need this, I didn't put it into my vsconfig config file and it automatically picked it up.  In fact if I do I get a complaint from VS Code.